flair/models/clustering/kmeans/k_Means.py

```diff
@@ -0,0 +1,84 @@
+import copy
+
+import numpy as np
+import torch
+
+from flair.datasets import DataLoader
+from flair.embeddings import DocumentEmbeddings
+from flair.models.clustering.clustering import Clustering
+from flair.models.clustering.distance import distance
+
+
+class KMeans(Clustering):
+    def __init__(self, k, embeddings: DocumentEmbeddings = None):
+        self.k = k
+        self.embeddings = embeddings
+        self.k_points = []
+        self.predict = []
```

Comment:
don't hold predictions as instance variable

Reply:
Should it be a return value of the algorithm ?